pkg/controller/eventprocessor/processor.go

```diff
@@ -98,6 +98,16 @@ func PersistPayloads(payloads []*interface{}) {
 			if err != nil {
 				log.Errorf("Error while persisting node %v", err)
 			}
+		} else if payload.ResourceType == "Namespace" {
```

Comment:
we are not listening for namespace events, can you please introduce the same.

Reply:
ok